src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/sealedclaim/SealedClaimResponseFormGeneratorForSpec.java

```diff
@@ -151,9 +150,7 @@ public SealedClaimResponseFormForSpec getTemplateData(CaseData caseData, String
     }
 
     private void addRepaymentPlanDetails(SealedClaimResponseFormForSpec.SealedClaimResponseFormForSpecBuilder builder, CaseData caseData) {
-        if (featureToggleService.isPinInPostEnabled() && isOneVOne(caseData)) {
```

Comment:
why these are being deleted? wouldnt we need 1v1 check ?

Reply:
We need this data for the 1v2 templates